Bottom: 7ecd0930049de4f1680657ab47b04462280e1719
Top:    edbdea4549c701c467994033715965a4c5547615
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-11-14 16:52:44 +0800

Refresh of tuntap-ndo_xdp_xmit-support

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index c53d297..cee170f 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -395,7 +395,7 @@ static void vhost_net_disable_vq(struct vhost_net *n,
 }
 
 static int vhost_net_enable_vq(struct vhost_net *n,
-			       struct vhost_virtqueue *vq)
+			        struct vhost_virtqueue *vq)
 {
 	struct vhost_net_virtqueue *nvq =
 		container_of(vq, struct vhost_net_virtqueue, vq);
